examples/example-protocol/src/assets/rust_wasmer_runtime_test/expected_types.rs

```diff
@@ -256,3 +256,10 @@ pub struct StructWithGenerics<T> {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub optional_timestamp: Option<MyDateTime>,
 }
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct StructWithOptions {
+    #[serde(default, skip_serializing_if = "String::is_empty")]
+    pub potentially_optional_string: String,
```

Comment:
I think this also needs one additional case: An `Option` value without the `skip_serializing_if` annotation. If I'm not mistaken, the currently generated TS types will assume that to be `undefined` too, though it would be serialized as `null`.

Reply:
I'll extend the struct with a bunch of different fields (some filled, some empty).